Slash/DB/MySQL/MySQL.pm

```diff
@@ -6209,23 +6209,25 @@ sub getCommentsForUser {
 	$other.= "ORDER BY $options->{order_col} $order_dir" if $options->{order_col};
 	$other.= " LIMIT $options->{limit}" if $options->{limit};
 
-	my $select = " cid, date, date as time, subject, nickname, "
+	my $select = " comments.cid, date, date as time, subject, nickname, "
 		. "homepage, fakeemail, users.uid AS uid, sig, "
 		. "comments.points AS points, pointsorig, "
 		. "tweak, tweak_orig, subject_orig, "
 		. "pid, pid AS original_pid, sid, lastmod, reason, "
 		. "journal_last_entry_date, ipid, subnetid, "
 		. "karma_bonus, "
-		. "len, badge_id, CONCAT('<SLASH type=\"COMMENT-TEXT\">', cid ,'</SLASH>') as comment";
+		. "len, badge_id, comment_text.comment as comment";
 	if ($constants->{plugin}{Subscribe} && $constants->{subscribe}) {
 		$select .= ", subscriber_bonus";
 	}
-	my $tables = "comments, users";
-	my $where = "sid=$sid_quoted AND comments.uid=users.uid ";
+	# Because fuck a bunch of doing search and replace to add comment text.
+	# That shit is insanely expensive compared to this.
+	my $tables = "comments LEFT JOIN comment_text ON comments.cid=comment_text.cid LEFT JOIN users ON comments.uid=users.uid";
+	my $where = "sid=$sid_quoted";
 
 	if ($cid && $one_cid_only) {
 		$where .= "AND cid=$cid";
-	} elsif ($user->{hardthresh} && !$options->{discussion2}) {
+	} elsif ($user->{hardthresh}) {
```

Comment:
Hrrm, this elsif looks like it may not be needed since we do thresholds in templates.  We need to check into this as well.